### PR TITLE
UI disable usage statistics by default and silent npm install

### PR DESCRIPTION
**Pull request by bogdan-sava**

### CODE REVIEW

File: open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/static/package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "Egeria UI application",
   "author": "Egeria contributors",
   "license": "Apache-2.0",
-  "repository": "odpi/egrias",
+  "repository": "odpi/egria",
```
Comment (planetf1) [inline open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/static/package.json:7]: is this a typo or intentional?

File: open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml [hunk 2]
```diff
@@ -358,7 +358,7 @@
                             <goal>npm</goal>
                         </goals>
                         <configuration>
-                            <arguments>install polymer-cli --prefix .</arguments>
+                            <arguments>install polymer-cli --silent --prefix .</arguments>
```
Comment (planetf1) [inline open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml:361]: What will this silent? I would worry if we hide any version/dependency warnings? Do you have a before/after comparison?

File: open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/static/package.json (hunk 1 above)
Comment (bogdan-sava) [reply open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/static/package.json:7]: typo! I fixed the only partial. Spotted the final 's' not the missing "e'

File: open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml (hunk 2 above)
Comment (bogdan-sava) [reply open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml:361]: this is silent the install of the polymer-cli. Always latest version.

Comment (bogdan-sava) [reply open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml:361]: But for dependencies (components) you are right, shouldn't be silent. I'll push the change.

Comment (bogdan-sava) [reply open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml:361]: During the build, node and npm are installed locally. 
Then polymer-cli (the silent one).
Then the dependencies according to the package.json from the static folder (silent removed).
Then polymer build is called.
Then the polymer build folder is copied to the resources.

Comment (planetf1) [reply open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml:361]: sounds good

Comment (planetf1) [reply open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml:361]: will approve with update :-)

Comment (bogdan-sava) [reply open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml:361]: not sure what 'with update' means :))

Comment (planetf1) [reply open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml:361]: I just meant when I saw your new commits - I think I replied just before you pushed the latest changes. I was just waiting for them. All looks good thanks

**Pull request merged**